src/loadOptions.js

```diff
@@ -97,6 +97,17 @@ export const loadOptions = function () {
     if (typeof(globalThis.vueSnapshots.formatting.selfClosingTag) !== 'boolean') {
       globalThis.vueSnapshots.formatting.selfClosingTag = false;
     }
+        
+    const whiteSpacePreservedOption = globalThis.vueSnapshots.formatting.tagsWithWhitespacePreserved;
+    if (Array.isArray(whiteSpacePreservedOption)) {
+      globalThis.vueSnapshots.formatting.tagsWithWhitespacePreserved = [...whiteSpacePreservedOption];
+    } else if (whiteSpacePreservedOption === false) {
+      globalThis.vueSnapshots.formatting.tagsWithWhitespacePreserved = [];
+    } else if (whiteSpacePreservedOption === true) {
+      globalThis.vueSnapshots.formatting.tagsWithWhitespacePreserved = true;
+    } else {
+      globalThis.vueSnapshots.formatting.tagsWithWhitespacePreserved = ['a', 'pre'];
+    }
```

Comment:
Adjusting the type checking and adding helpful messages if they pass in the wrong stuff.

```suggestion
    const preserveWhitespaceMessage = [
      'vueSnapshots.formatting.tagsWithWhitespacePreserved',
      'must an Array of tag names, like [\'a\' ,\'pre\'],',
      'or a boolean for all tags, or no tags.'
    ].join(' ');
    if (Array.isArray(whiteSpacePreservedOption)) {
      const justStrings = whiteSpacePreservedOption.filter(function (tags) {
        return typeof(tag) === 'string';
      });
      if (whiteSpacePreservedOption.length !== justStrings.length) {
        logger(preserveWhitespaceMessage);
      }
      globalThis.vueSnapshots.formatting.tagsWithWhitespacePreserved = justStrings;
    } else if (typeof(whiteSpacePreservedOption) !== 'boolean') {
      if (whiteSpacePreservedOption !== undefined) {
        logger(preserveWhitespaceMessage);
      }
      globalThis.vueSnapshots.formatting.tagsWithWhitespacePreserved = ['a', 'pre'];
    }
```